esp32/PermanentMarker36.c

```diff
@@ -0,0 +1 @@
+../../components/ugfx/PermanentMarker36.c
```

Comment:
I did this only because that's how the others were setup. It seems to just work when I remove the symlinks and just move the fonts from [components/ugfx/](https://github.com/SHA2017-badge/Firmware/tree/master/components/ugfx) to [micropython/esp32/](https://github.com/SHA2017-badge/Firmware/tree/master/components/ugfx). I could commit that to both PR's if desired.